addons/l10n_pe/data/res_country_data.xml

```diff
@@ -7,24 +7,21 @@
         <field name="arch" type="xml">
             <form>
                 <div class="o_address_format">
-                    <field name="parent_id" invisible="1"/>
-                    <field name="type" invisible="1"/>
-                    <field name="street_name" placeholder="Street Name..." attrs="{'readonly': [('type', '=', 'contact'),('parent_id', '!=', False)]}" class="oe_read_only"/>
-                    <field name="street" placeholder="Street" class="oe_edit_only"/>
-                    <field name="street2" placeholder="Street2" invisible="1"/>
-                    <div class="o_row">
-                        <label for="street_number" class="oe_edit_only"/>
-                        <field name="street_number" attrs="{'readonly': [('type', '=', 'contact'),('parent_id', '!=', False)]}"/>
+                    <field name="street" placeholder="Street..." class="o_address_street oe_read_only"/>
+                    <div class="oe_edit_only o_row">
+                        <field name="street_name" placeholder="Street" style="flex: 3 1 auto"/>
                         <span> </span>
-                        <label for="street_number2" class="oe_edit_only"/>
-                        <field name="street_number2" attrs="{'readonly': [('type', '=', 'contact'),('parent_id', '!=', False)]}"/>
+                        <field name="street_number" placeholder="House #" style="flex: 1 1 auto"/>
+                        <span> - </span>
+                        <field name="street_number2" placeholder="Door #" style="flex: 1 1 auto"/>
```

Comment:
District and city is a bit uncertain for the moment, but we do not need to split the street anywhere, so we can simplify this view and simply remove it.

Reply:
done